bashrc.d/zz-10-base.sh

```diff
@@ -344,10 +344,16 @@ fi
 if [[ ${TERM:-} == dumb ]]; then
     export EDITOR=$(type -p emacsclient)
     export NODE_NO_READLINE=1
-    export PAGER=cat
+    export PAGER=$(type -p cat)
```

Comment:
Were you somewhere that didn't have `cat`?

Reply:
nvm see your comment about crontab.

Reply:
It's just better to have absolute paths for this type of var.